.prettierignore

```diff
@@ -8,4 +8,6 @@
 **/.tsbuild*
 **/.next/*
 *.mdx
-**/_archive/*
\ No newline at end of file
+**/_archive/*
+**/*.json
+**/*.js
```

Comment:
not sure if this is from steves or jamies changes but we generally do want to include .json & .js files in prettier! we've had issues before with unrelated formatting changes getting into diffs with these files because some team members don't have prettier configured on save. Could we keep these included? Are there specific files causing problems?

Reply:
Fixing!